src/cosmos.ts

```diff
@@ -303,6 +324,20 @@ export class CosmosWrapper {
     }
   }
 
+  async queryDenomsMetadata(): Promise<DenomMetadataResponse> {
+    try {
+      const req = await axios.get<DenomMetadataResponse>(
+        `${this.sdk.url}/cosmos/bank/v1beta1/denoms_metadata`,
```

Comment:
I can see cosmos-sdk denoms_metadata utilize pagination functionality, and i think we need to add pageRequest params into the library.